rosette/rbl/rosette/tests/arithmetic_test.rbl

```diff
@@ -0,0 +1 @@
+(if (match? [(+ 2 8) (+ (+ (+ (+ (+ (+ (+ (+ (+ 1 2) 3) 4) 5) 6) 7) 8) 9) 10) (+ 2147483647 2147483647) (+ 2.5 7.5) (+ 3.141592653589793 3.141592653589793) (- 47 37) (- (- 2147483647) 1) (* 2 5) (* (- 5) (- 2)) (* 2147483647 2147483647) (/ 120 12) (/ 2147483647 (- 1)) (+ (- (- (+ (- 10) (* 4 5)) (/ 30 10)) 2) 5) (+ 10 (* (- 10 10) 10)) (- (+ (- 10.5 0.5) (/ 2.5 2.5)) 1.0)] [10 55 -2 10.0 6.283185307179586 10 0 10 10 1 10 1 10 10 10.0]) (seq (ostream-print (ostream-new "results.txt") "Pass") (display #\\n)) (seq (ostream-print (ostream-new "results.txt") "Fail") (display #\\n)))
```

Comment:
Any reason not to format this nicely?

Reply:
Whoops, thanks for the catch. That shouldn't be checked in.